airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -112,7 +112,7 @@ def set_working_directory_to_root() -> None:
     os.chdir(working_dir)
 
 
-def get_modified_files(
+async def get_modified_files(
```

Comment:
❓ why did we have to make this async?

Reply:
We didn't _have_ to, but we were able to! (In order to remove the `anyio.run`)

Reply:
E.g. [here](https://github.com/airbytehq/airbyte/pull/31748/files#diff-f35c71638f2ffaabb83f6f5b5aa4c5c8ef5e2baab9968e7aa14fc32c9a7f54d2R112)

Reply:
Good call!